scripts/webpack.js

```diff
@@ -1,5 +1,6 @@
 const path = require('path');
 const webpack = require('webpack');
+const TerserPlugin = require("terser-webpack-plugin");
```

Comment:
Note to me: this should be added as an explicit dependency, it's only available because of yarn/lerna hoisting.